templatestore/views.py

```diff
@@ -553,6 +553,7 @@ def get_config_view(request):
                     tes[t.type] = {
                         "sub_type": [{"type": t.sub_type, "render_mode": t.render_mode}]
                     }
+                    tes[t.type]["attributes"] = json.loads(t.attributes)
```

Comment:
this can be added in above dict, way `sub_type` is added

Reply:
that would result into repetition of same thing again and again in each sub_type. This will also help in easy access of these attributes in frontend code.